#!/usr/bin/env python
PACKAGE = "projects_robotics"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("computation_type",    int_t,    0, "0 Differential Drive, 1 Ackermann", 0,  0, 1)
gen.add("reset_signal", int_t, 0, "change to reset", 0, 0, 1)
gen.add("change_coordinates_signal", int_t, 0, "switch to change coordinates", 0,0,1)
gen.add("x_coordinate", double_t, 0, "New X", 0)
gen.add("y_coordinate", double_t, 0, "New Y", 0)



exit(gen.generate(PACKAGE, "projects_robotics", "dynamic_ric"))